### ISSUE TITLE
Update React dependencies

### ISSUE THREAD

**Issue by cschleiden** [Microsoft/vss-web-extension-sdk#108]
We should update the React dependencies provided to extensions

**Comment by nkirchem** [comment]
@cschleiden No, this is pretty much deprecated in favor of:
https://github.com/Microsoft/azure-devops-extension-sdk

Which we just added and will be finishing out, with examples, over this sprint and next.

**Comment by nkirchem** [comment]
@pabrams Yes, we have to go through a legal review before new OSS-dependent repositories are available publicly, and I'm waiting on an approval. It'll be visible once we are ready with our new infrastructure.

**Comment by pabrams** [comment]
@nkirchem  Will azure-devops-extension-sdk be a new sdk, or is it really just a re-brand of the vsts-web-extension-sdk?  I'm in the middle of trying to figure out how to get some react packages to work with the vss-web-extension-sdk in a fork of an older extension, and wondering if I should just give up and wait for this new repo.  Also, will it be more for the Azure side of things, or will it still be useful for our installation of TFS 2017 U3?

**Comment by pabrams** [comment]
@nkirchem @cschleiden Should my organization just give up on trying to use these web extensions?  Can you please tell us how to use this sdk with modern versions of typescript and react, or tell us when we can start using azure-devops-extension-sdk?
